### DISCUSSION TITLE
How to remove Cordova 12 splash screen from the app?

### DISCUSSION THREAD

**Discussion by ghevge** [apache/cordova#401] [Q&A]
I've just installed Cordova 12 and I've noticed that there is now a default cordova splash screen being injected automatically into the app.
My question is: How can I remove it?
Thanks

**Comment by breautek** [comment]
The splashscreen cannot be removed as Android will inject a splashscreen itself if the app doesn't explicitly configure one.

You can however use the [AndroidWindowSplashScreenAnimatedIcon](https://cordova.apache.org/docs/en/11.x/core/features/splashscreen/index.html#androidwindowsplashscreenanimatedicon) preference to set the splashscreen icon.

For context, Android 12 (API 31) introduced OS-level splashscreens which all apps will forcefully have when running on Android 12 and later devices. Pre-Android 12/API 31 devices splashscreens were entirely controlled by the app.

In order to have a consistent splashscreen behaviour across all devices, we (Apache) use Google's splashscreen compatibility library to bring Android 12 splashscreen system into older devices. This is implemented as of cordova-android@11. Due to how Android works, the splashscreen cannot be removed.

I think iOS requires a "storyboard' style splashscreen as well, which can be set using a single image file: https://cordova.apache.org/docs/en/11.x/core/features/splashscreen/index.html#single-image-launch-screen
